geopandas/sindex.py

```diff
@@ -1,3 +1,7 @@
+"""Spatial index."""
+
+from __future__ import annotations
```

Comment:
why did this get added in?, it does look like annotations were changed.

Reply:
Not sure. I don't recall typing it (and wouldn't know why), so must have been an automatic fix from somewhere?

Removed it again.